modules/exploits/freebsd/local/freebsd_fd_ref_count_priv_esc.rb

```diff
@@ -0,0 +1,266 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = AverageRanking
```

Comment:
Maybe should be `ManualRanking` due to the chances of rebooting and/or bricking the system?

Reply:
No issues with that at all.  Will wait to change it until everything else is reviewed.